site/_data/contributors.yml

```diff
@@ -205,6 +205,12 @@
   githubId: laurentgo
   org: Dremio
   role: PMC
+- name: Lei Shen
+  apacheId: lakeshen
+  githubId: LakeShen
+  pronouns: he/him
+  org:
```

Comment:
Are there any omissions?

Reply:
> Are there any omissions?

Hi,I don't write the `org`,it's ok for me.